compositor_render/src/transformations/web_renderer/chromium_sender_thread.rs

```diff
@@ -161,42 +173,30 @@ impl ChromiumSenderThread {
 
 struct ThreadState {
     browser: cef::Browser,
-    shared_memories: HashMap<(NodeId, usize), Shmem>,
+    shared_memory: HashMap<NodeId, Vec<SharedMemory>>,
```

Comment:
I wasn't a fan of `shared_memories` name but the plural form helped distinguishing a collection of shared memory from a single shared memory. On the other hand I can't think of better name 😅